spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -1014,7 +1013,7 @@ _With a body_
 </c:out>
 
 .*Body Content*
-JSP. The Jakarta Server Pages container processes the body
+Jakarta Server Pages. The Jakarta Server Pages container processes the body
```

Comment:
Should we use the singular for these since it was singular previously? Perhaps Jakarta Server Pages page?